website/docs/docs/dbt-cloud/access-control/access-control-overview.md

```diff
@@ -0,0 +1,120 @@
+permissions of users on their account. This access control comes in two
+flavors: License-based, and Role-based.
+
+- **License-based Access Controls:** User are configured with account-wide
+  license types. These licenses control the specific parts of the dbt Cloud application
+  that a given user can access.
+- **Role-based Access Control (RBAC):** Users are assigned to _groups_ that have
+  specific permissions on specific projects or the entire account. A user may be
+  a member of multiple groups, and those groups may have permissions on multiple
+  projects.
+
+## License-based access control
+
+Each user on an account is assigned a license type when the user is first
+invited to a given account. This license type may change over time, but a
+user can only have one type of license at any given time.
+
+A user's license type controls the features in dbt Cloud that the user is able
+to access. dbt Cloud's two license types are:
+ - **Read Only**
+ - **Developer**
+
+For more information on these license types, see [Seats & Users](cloud-seats-and-users).
+At a high-level, Developers may be granted _any_ permissions, whereas Read Only
+users will have read-only permissions applied to all dbt Cloud resources
+regardless of the role-based permissions that the user is assigned.
+
+## Role-based access control
+
+Role-based access control allows for fine-grained permissioning in the dbt Cloud
+application. With role-based access control, users can be assigned varying
+permissions to different projects within a dbt Cloud account. For teams on the
+Enterprise tier, role-based permissions can be generated dynamically from
+configurations in an [Identity Provider](sso-overview).
+
+Role-based permissions are applied to _groups_ and pertain to _projects_. The
+assignable permissions themselves are granted via _permission sets_.
+
+
+### Groups
+
+A group is a collection of users. Users may belong to multiple groups. Members
+of a group inherit any permissions applied to the group itself.
+
+Users can be added to a dbt Cloud group based on their group memberships in the
+configured [Identity Provider](sso-overview) for the account. In this way, dbt
+Cloud administrators can manage access to dbt Cloud resources via identity
+management software like Azure AD, Okta, or GSuite. See _SSO Mappings_ below for
+more information.
+
+You can view the groups in your account or create new groups from the **Team > Groups**
+page in your Account Settings.
+
+<Lightbox
+    src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/group-list.png"
+    title="Viewing a list of groups in the Account Settings page."
+/>
+
+
+### SSO Mappings
+
+SSO Mappings connect Identity Provider group membership to dbt Cloud group
+membership. SSO Mappings are declared as a list of IdP group names within a
+specific dbt Cloud group. When a user logs into dbt Cloud via a supported
+identity provider, their IdP group memberships are synced with dbt Cloud. Upon
```

Comment:
```
SSO Mappings connect Identity Provider group membership to dbt Cloud group
membership. SSO Mappings are declared as a list of IdP group names within a
specific dbt Cloud group.
``` 

These two sentences are a tad confusing (I think it's more because of the passive voice in the second sentence). Perhaps the second sentence can read more like this:

``` 
By declaring a list of IdP group names related to a specific dbt Cloud group, you will make the relationship of those IdP Groups to the dbt Cloud group.

Reply:
I ended up just deleting this sentence - I don't really think it adds anything here! Good call :)